lib/modules/platform/gitlab/index.spec.ts

```diff
@@ -2539,6 +2539,40 @@ describe('modules/platform/gitlab/index', () => {
         })
       ).toResolve();
     });
```

Comment:
Do we need to update the `addLabels` and `labels` config option docs? Those docs mention Renovate's default behavior.

Reply:
Yes